nss-day-cohort-48/Rare-API-Django-Wandering-Wizards
# Ajw add remove tags

#### Changes Made

1. Modified file `post.py` in views to add tags in create and update functions.

#### Steps to Review
1. Checkout to this branch locally
   ```
   git fetch --all
   git checkout ajw-add-remove-tags
   ```

2. Ensure the backend and client are both running.

4. Test app functionality.
   
   > Example: Run http://localhost:8000/posts in postman and see if tags populated post
   > 

5. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
